packages/api/src/schema/api-schema.yaml

```diff
@@ -1763,8 +1810,10 @@ components:
             properties:
               eventName:
                 type: string
+                example: event name
```

Comment:
This description is literally the same words as the field itself xD

Reply:
That was me just setting the example so that we didn't get 'null' coming out in the docs